shipstation_integration/shipments.py

```diff
@@ -158,3 +156,35 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+def create_shipment(shipment, delivery_note, store):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.description_of_content = "Farm plastic"
+	shipment_doc.value_of_goods = delivery_note.total
+	shipment_doc.shipment_id = shipment.shipment_id
+	shipment_doc.pickup_date = shipment.create_date
+	shipment_doc.carrier = shipment.carrier_code.upper()
+	shipment_doc.carrier_service = shipment.service_code.upper()
+	shipment_doc.awb_number = shipment.tracking_number
+	shipment_doc.pallets = "No"
+	shipment_doc.service_provider = "Shipstation"
+	shipment_doc.incoterm = "DAP (Delivered At Place)"
+	shipment_doc.shipstation_store_name = store.store_name
+	shipment_doc.shipstation_order_id = shipment.order_id
+	shipment_doc.marketpalce = store.marketplace_name
+	shipment_doc.marketplace_order_id = shipment.order_number
+	shipment_doc.flags.ignore_mandatory = True
+	shipment_doc.flags.ignore_validate = True
+	shipment_doc.append("shipment_parcel", {
+		"length": shipment.dimensions.length,
+		"width": shipment.dimensions.width,
+		"height": shipment.dimensions.height,
+		"weight": shipment.weight.value
+	})
```

Comment:
Do a check for `shipment.dimensions` before adding this one, in case there's no dimensions for a shipment somehow.